reference/cpp-api.md

```diff
@@ -373,8 +373,8 @@ The following tables describe the C++ classes and their methods.
 | &nbsp;&nbsp; bool [isPointCloudEnabled](lidar.md#wb_lidar_enable_point_cloud)();                                                     |
 | &nbsp;&nbsp; const float * [getRangeImage](lidar.md#wb_lidar_get_range_image)() const;                                               |
 | &nbsp;&nbsp; const float * [getLayerRangeImage](lidar.md#wb_lidar_get_range_image)(int layer) const;                                 |
-| &nbsp;&nbsp; const [WbLidarPoint](lidar.md#wblidarpoint) * [getPointCloud](lidar.md#wb_lidar_get_point_cloud)() const;               |
-| &nbsp;&nbsp; const [WbLidarPoint](lidar.md#wblidarpoint) * [getLayerPointCloud](lidar.md#wb_lidar_get_point_cloud)(int layer) const; |
+| &nbsp;&nbsp; const [LidarPoint](lidar.md#wblidarpoint) * [getPointCloud](lidar.md#wb_lidar_get_point_cloud)() const;                 |
```

Comment:
No, forget it, I didn't see it well due to the Github diff formating, this is correct.